### PR TITLE
Make Obo compatible with PHP7.2

### PR DESCRIPTION
**Pull request by ZZromanZZ**

### CODE REVIEW

File: src/Carriers/EntitiesCollection.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ protected function &variables(array $requiredItems = null) {
             foreach ($requiredItems as $key => $requiredItem) if (isset($variables[$requiredItem])) unset($requiredItems[$key]);
         }
 
-        if (!$this->entitiesAreLoaded AND ((\count($requiredItems) !== 0) OR $requiredItems === null)) {
+        if (!$this->entitiesAreLoaded AND ($requiredItems === null OR \count($requiredItems) !== 0)) {
```
Comment (hAdamm) [inline src/Carriers/EntitiesCollection.php]: maybe better `if (!$this->entitiesAreLoaded AND ($requiredItems === null OR !$requiredItems))`

File: .travis.yml [hunk 2]
```diff
@@ -19,11 +20,8 @@ before_script:
     # update Composer to current version, install Nette Tester & Code Checker
     - composer self-update
     - composer install --no-interaction --prefer-source
-    - composer create-project nette/code-checker code-checker 2.5 --no-interaction --prefer-source
```
Comment (ZZromanZZ) [inline .travis.yml]: There is problem with Nette Code-checker. Version > 2.5 is ready for PHP 7.2 but coding standard is not usable for Obo. I temporarily removed this here just for tests to pass, but this is not final solution.

File: src/Carriers/EntitiesCollection.php (hunk 1 above)
Comment (ZZromanZZ) [reply src/Carriers/EntitiesCollection.php]: Ok, condition was changed only because parameter to count() function has to be array, not null in this case.

Comment (ZZromanZZ) [reply src/Carriers/EntitiesCollection.php]: The condition changed.

**Pull request merged**